src/services/user-service.js

```diff
@@ -20,8 +20,11 @@ const setProfile = (userId, ...fields) => {
  * @param {string} userId id of the user
  * @return {object} a promise
  */
-const getProfile = (userId) => {
-  return firebase.firestore().collection(USERS_COLLECTION).doc(userId).get();
+const findUser = (userId) => {
+  return ALL_USERS.doc(userId)
+    .get().then((res) => {
```

Comment:
We should not be handling the promise here. Any errors returned by Firestore will be ignored since our return variable is now the data and not the promise.

Returning the promise will enable us to retain error handling in the implementing files (the places where we have `setError(error)` commented out)

Reply:
Oh I see; did not realize that was the reasoning. Will update.